src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+          caption="Don't use alerts to highlight the outcomes of user actions."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-6.png"
+          caption="Do use toasts to call out the outcomes of user actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/alert/alert-anatomy.png"
+      >
+        <sky-docs-anatomy-item>
+          Alert icon
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>
+          Text
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item
+          isOptional="true"
+        >
+          Close button
+        </sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      
+        <sky-docs-design-guideline
+        heading="Alert type"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-1.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Danger alert – for errors or to let the user know the system is in an unusable state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-2.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Info alert – for messages that provide helpful information or context but do not require action.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-3.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Success alert – for messages conveying a positive or successful state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-4.png"
```

Comment:
In this image, I'd tweak the text to make it a complete sentence: "This gift is more than 5 days old but is not acknowledged."